tasks/main.yaml

```diff
@@ -60,6 +62,32 @@
   tags:
     - zookeeper_dirs
 
+- name: Remove directory of ZooKeeper application logs
```

Comment:
This breaks the idempotent nature of this. Each run is deleting this dir and then recreating the symlink further down so always comes up with 2 changes. Is this task to clean up from previous versions of this role prior to your changes where this `/usr/share/zookeeper/log` directory would have existed previously? This would also mean that all application logs would be deleted for any existing installation where this was run. I agree that `/var/log/zookeeper` is preferable, just wanting to consider options for mitigating this.

Reply:
Apologies, please ignore the second part of this comment. This role didn't used to create a `/usr/share/zookeeper/log` directory previously so there shouldn't be any loss of data. The first part of the comment should still be accurate however, this is deleting and re-adding the directory symlink so is not idempotent.

Reply:
How should we handle this? As the directory logs inside the unpacked zookeeper will always exists.

Reply:
I'm a little confused now. The role already creates the `/var/log/zookeeper` directory and this is set as the value in the `ZOO_LOG_DIR` environment variable value, https://github.com/sleighzy/ansible-zookeeper/blob/master/templates/default.j2#L1, which ZooKeeper uses for it's logs when it starts up. There shouldn't be any log dirs in the unpacked ZooKeeper installation directory. If you are wanting to have a symlink to the `/var/log/zookeeper` dir under `/usr/share/zookeeper` dir then you can leave that single task there for just adding the link.

Reply:
You're right, there is no logs folder inside the tarball. I must have made some previous tests and it did create one and assumed that wrongly.